rules/windows/registry_event/sysmon_stickykey_like_backdoor.yml

```diff
@@ -39,13 +39,15 @@ logsource:
     product: windows
 detection:
     selection_process:
-        ParentImage:
-            - '*\winlogon.exe'
-        CommandLine:
-            - '*cmd.exe sethc.exe *'
-            - '*cmd.exe utilman.exe *'
-            - '*cmd.exe osk.exe *'
-            - '*cmd.exe Magnify.exe *'
-            - '*cmd.exe Narrator.exe *'
-            - '*cmd.exe DisplaySwitch.exe *'
+        ParentImage|endswith:
+            - '\winlogon.exe'
+        CommandLine|contains|all:
+            - 'cmd.exe'
```

Comment:
I believe that it could be changed to a `ProcessName`

Reply:
Fixed with Image